scripts/apps-version-check.sh

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-latest_release=$(git describe --abbrev=0 --tags)
+## compaare to the latest release version tag:
+## but do not include alpha, beta and rc versions
+latest_release="$(git describe --abbrev=0 --tags --match 'v*' --match 'e*' --exclude '*beta*' --exclude '*alpha*' --exclude '*rc*')"
```

Comment:
Just to confirm: it's ok to mix `v*` and `e*` releases because, currently, they live in disjoint repos, right?

Reply:
for 5.0, they will be in the same repo

Reply:
I see.  Assuming that we always release both community and enterprise at the same time, then there shouldn't be much trouble if either one is retrieved as the latest.  

(maybe unless an EE app is being analyzed below, and the latest release retrieved is CE? :thinking: )

Reply:
we may not always release ce and ee at the same commit.

but whenever a release is cut, regardless of ce or ee, the erl file changes after this tag should have a corresponding app vsn bump.

e.g.
this might be the case `commit1, commit2(tag v5.0.0), commit3, commit4(tag e5.0.0)`

changes in `commit3` and `commit4` should force vsn bumps